nodes/src/nodes/tool_firecrawl/IInstance.py

```
@@ -26,23 +26,128 @@
+                'content': {'type': 'string'},
+                'metadata': {'type': 'object'},
+            },
+        },
+        description='Scrape a single web page and return its content.',
+    )
+    def scrape_url(self, args):
+        """Scrape a single web page."""
+        args = _normalize_tool_input(args)
+        url = args.get('url')
+        if not url:
+            raise ValueError('scrape_url requires a `url` parameter')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)